test/sdf/unrecognized_elements_with_namespace.sdf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <model name="joint_incorrect_tags" some_attribute="staheousth">
```

Comment:
nit: what's `staheousth`?

Reply:
Just some arbitrary text content. Changed to "some value"